### PR TITLE
ES6 Project

### PR DESCRIPTION
**Pull request by Ashnaali3255**
- Use ES6 modules to write modular JavaScript.
- Learn how to use proper ES6 syntax.
- Understand improvements that ES6 brings to JavaScript.
- Use npm as a software package system.

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Ashnaali3255,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### When I start your app it just keeps refreshing (see video) and I am unable to do anything on the page. Kindly look into this.

  [screen-capture.webm](https://user-images.githubusercontent.com/36542986/220351217-6e755328-1c72-4e21-87ed-840533209bcb.webm)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Ashnaali3255** [comment]
@TedLivist   Can you help me what i have to do for this please help me.....🙏🏻🙏🏻🙏🏻

File: index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const bookCollection = new BookCollection();
+document
+  .getElementById('add-button')
+  .addEventListener('click', bookCollection.addBook);
+bookCollection.displayBooks();
+
+const now = DateTime.now();
+const date = now.toLocaleString(DateTime.DATETIME_FULL);
+
+document.getElementById('date').textContent = date;
```
Comment (AbayomiOlaoye) [inline index.js:13]: - [ ] [OPTIONAL] Your code is really impressive and the design is requisite! 💯 🥇 but you can also improve user experience by making your time to update in real-time just like in normal setting. without having to refresh. For this, I think [ `setInterval()`](https://www.w3schools.com/jsref/met_win_setinterval.asp)method will be a great tool. :; 💪

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @Ashnaali3255 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI Interface :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- All functions working well ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**